Dockerfile

```diff
@@ -7,13 +6,19 @@ COPY requirements.txt /code/requirements.txt
 
 RUN pip install --no-cache-dir -r /code/requirements.txt
 
-COPY ./app /code/app
+COPY ./app /code/app 
 
 # Local development key set
 # ENV TYPES: dev, production
 # When set to dev, API Key on endpoint requests are just 'dev'
 # When set to production, API Key on endpoint requests are the actual API Key
 
-ENV PYTHONPATH=/code/app
+<<<<<<< HEAD
```

Comment:
Can you remove this, I think you added my accident @EdwardShiung

Reply:
sure!! haha thanks man!!

Reply:
I fixed it!